rules/windows/credential_access_potential_lsa_memdump_via_mirrordump.toml

```diff
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2021/09/27"
+maturity = "production"
+updated_date = "2021/09/27"
+min_stack_version = "7.12.0"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a suspicious process access event by LSASS where the granted access value is equal to duplicate process handle
+and from an unknown call trace module. This may indicate an attempt to dump Lsass memory using MirrorDump tool in preparation
+for credential access.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-windows.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Potential Credential Access via MirrorDump"
+references = ["https://github.com/CCob/MirrorDump"]
+risk_score = 47
+rule_id = "02a4576a-7480-4284-9327-548a806b5e48"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Credential Access"]
+timestamp_override = "event.ingested"
+type = "query"
+
+query = '''
+event.category:process and event.code:"10" and process.name :"lsass.exe" and winlog.event_data.GrantedAccess :"0x40" and
```

Comment:
I wouldn't be opposed to this being in EQL to include the comments in the query from the issue

Reply:
I noticed you are not referencing the zipped file in a sequence, is that because it was too hard to link them to a single process? If so, we could change the name and description of this to be a bit more general, like `Potential Credential Access via DuplicateHandle in LSASS`

Reply:
it will require to log every file creation (for Sysmon users) because the zip file name is a controlled parameter. I like the suggested name `Potential Credential Access via DuplicateHandle in LSASS`.